core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -165,4 +168,5 @@ object JsoupBrowser {
     def toHtml = underlying.outerHtml
   }
 
+  private val baseUri = ""
```

Comment:
This seems a bit out of place... why did you have to specify a base URI? What consequences does it have in Jsoup parsing?

If we really want/need to have this variable, please move it to the top of the file together with the other private vals.

Reply:
In this case, we are parse from text and need a `baseUri` here.

Just like the former version of `parseInputStream` did

```
  def parseInputStream(inputStream: InputStream, charset: String): JsoupDocument =
    using(inputStream) { _ => JsoupDocument(Jsoup.parse(inputStream, charset, "")) }
    // here, "" stands for the baseUri parameter 
```

At first, I planned to place this line of code next to other private variables, but after observing, I couldn't find the previous private variables.

I have changed the line to the top of companion object.